<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevaObra</class>
 <widget class="QDialog" name="dlgNuevaObra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>621</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLineEdit" name="txtIdioma">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Idioma</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTipo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFormato">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Formato</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spTomos">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Tomos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNumeroPags">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Número pags.</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAutor">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Autor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Introducir autor</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Impresión</string>
    </attribute>
    <widget class="QCheckBox" name="ckImpreso">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Impreso</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckTalVezImpreso">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tal vez impreso</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckManuscrito">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Manuscrito</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLugarOriginalImpresion">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lugar original de impresión</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbLugarImpresion">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lugar de impresión</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1400</number>
     </property>
     <property name="maximum">
      <number>2000</number>
     </property>
    </widget>
    <widget class="QLabel" name="spFechaImpresion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>58</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string>Fecha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEditor">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Editor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btAnadirLugar">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Extras</string>
    </attribute>
    <widget class="QCheckBox" name="ckReferencias">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Referencias</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckTituloReducido">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Título reducido</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckContenido">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Contenido</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckDudoso">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Dudoso</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckExpurgable">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>84</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Expurgable</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spInteresante">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Interesante</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spFiabilidad">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>48</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Fiabilidad</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Notas</string>
    </attribute>
    <widget class="QTextEdit" name="txtNotas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>461</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
